test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -143,6 +142,125 @@ function GetTestData
     return $body
 }
 
+<#
+    Extract the raw content from a response and
+    convert the embedded json to PSObjects.
+    Assumes the raw content is from the  HTTPListener.
+#>
+function Convert-ResponseRawContent
+{
+    param
+    (
+        [Parameter(Mandatory)]
+        [ValidateNotNull()]
+        [Microsoft.PowerShell.Commands.WebResponseObject] $Response
+    )
+
+    $index = $Response.RawContent.IndexOf("{")
```

Comment:
Looking at your usage below, you pass in the output of Invoke-WebRequest to this helper.  This helper isn't necessary.  Line 189 below can just be:

```powershell
$result.Content = $result.Output.Content | ConvertFrom-Json
```

This is assuming the output you create in httplistener is json (and if not, perhaps it should be).